storage/src/tests/persistence/common/persistenceproviderwrapper.h

```diff
@@ -61,12 +64,16 @@ class PersistenceProviderWrapper : public spi::AbstractPersistenceProvider
      * return the given error without the wrapped SPI ever being invoked.
      */
     void setResult(const spi::Result& result) {
+        Guard guard(_lock);
         _result = result;
     }
     void clearResult() {
         _result = spi::Result(spi::Result::ErrorType::NONE, "");
     }
-    const spi::Result& getResult() const { return _result; }
```

Comment:
clearResult() above doesn't seem to be used. Consider removing it.

Reply:
Good catch, done.